src/org/usfirst/frc4904/robot/commands/SetRampState.java

```diff
@@ -0,0 +1,18 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.GearIO.RampState;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class SetRampState extends Command {
+	public SetRampState(RampState state) {
+		RobotMap.Component.gearIO.setRampState(state);
```

Comment:
This will change the ramp state when you _construct_ the command, not _start_ it.